apps/vor/.vscode/settings.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
This shouldn't be committed. Let's keep this codebase IDE agnostic for now. We should also add `.vscode` folder to the gitignore.